EMBRACE_Client/ConditionSetup.m

```diff
@@ -0,0 +1,21 @@
+//
+//  ConditionSetup.m
+//  EMBRACE
+//
+//  Created by Jonatan Lemos Zuluaga (Student) on 10/24/14.
+//  Copyright (c) 2014 Andreea Danielescu. All rights reserved.
+//
+
+#import "ConditionSetup.h"
+
+@implementation ConditionSetup
+    @synthesize condition;
+    @synthesize language;
+
+- (id) init {
+    condition = @"Embrace";
```

Comment:
Might be better to use typedef enums instead of string values. (See line notes in ConditionSetup.h.)